src/test/java/calculator/app/MainTest.java

```diff
@@ -0,0 +1,81 @@
+package calculator.app;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MainTest {
+
+    private static Main main;
+
+    @BeforeAll
+    static void setMain() {
+        main = new Main();
+    }
+
+    @Test
+    @DisplayName("Test addition functionality")
+    void givenTwoNumbers_whenAdditionFunctionality_thenCorrectResult() {
+        //given
+        Integer a = 10;
+        Integer b = 5;
+        Integer expectedResult = 15;
```

Comment:
expected / actual is often used in tests.

Integer expected = 15; 

//when
Integer actual = main.addition(a, b);